Sources/Plasma/FeatureLib/pfPython/pyEnum.cpp

```diff
@@ -360,21 +299,21 @@ PYTHON_TYPE_START(EnumValue)
     nullptr,                            /* tp_getattr */
     nullptr,                            /* tp_setattr */
     nullptr,                            /* tp_as_async */
-    EnumValue_repr,                     /* tp_repr */
+    (reprfunc)EnumValue_repr,           /* tp_repr */
```

Comment:
IIRC even with a cast, this will trigger a warning on newer versions of GCC...

Reply:
Thanks GCC. So much for avoiding dumb casts 😞